rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

```diff
@@ -110,37 +115,131 @@ Remove automation projects you no longer need. This cannot be undone and is neit
+
+Click *Back to Process* to return to or open the project.
+
+[[form-createprocessboardtask]]
+=== Create a Task
+
+Create a process project task to enhance the automation.
+
+Examples of such tasks could be `Design: Leaning (Remove superfluous elements from the graph, check for dead ends and double doors)`, `Add missing process documentation (Prepare decision table)` or `Kick-off Meeting`.
+
+To create a task:
+
+. <<openprocessboard, Open a process board>>.
+. Click *+ Create Task* in the *To-do* column.
+. Complete the form *Create new task*:
+* *Task Owner*:
++
+The task owner is responsible for completing the task. Choose a user from the process team. The task item displays the user's profile picture. RPA Manager shows all tasks of the task owners on their personal *My Process Board* view in the *My RPA* module.
+* *Priority*:
++
+You can set the priority of the task to low, medium, or high. The task item displays a grey, yellow or red label, respectively.
+* *Due date*:
++
+Date on which the task should be completed.
+* *Effort*:
++
+Time the task will approximately take.
+. Click Save.
+
+=== Edit a Task
+
+Edit tasks to reflect changed requirements:
+
+. <<openprocessboard, Open a process board>>.
+. Click a task item.
+. Edit the form *Edit task*.
++
+For an explanation of the properties, see <<form-createprocessboardtask, *Create a task*>>.
+. Click *Save*
+
+=== Delete a Task
+
+Delete obsolete tasks:
+
+. <<openprocessboard, Open a process board>>.
+. Click a task item.
+. Click *Delete*.
+. Confirm the deletion.
+
+=== Move a Task
+
+Move tasks within a column to sort them or across columns to reflect their progress:
+
+. <<openprocessboard, Open a process board>>.
+. Drag a task item to its new place.
+
+If you have enabled the creation of automatically generated tasks for the phase transitions, move this task to the *Completed* column after all other tasks have been completed to enable the transition of the project to the next phase.
+
 == Analyze the Financial Aspects of a Process
 
-Open the xref:processoperations-financeanalysis.adoc[view *Finance Analysis*] to record, monitor, and analyze the financial aspects of a process:
+Open the xref:processoperations-financeanalysis.adoc[*Finance Analysis* view] to record, monitor, and analyze the financial aspects of a process:
 
-. In a category in the *Process Management* view of the *Process Automation* module, click the *Finance Analysis* icon image:financeanalysis-icon.png[ symbol,1.5%,1.5%] the panel of the project you want to analyze.
+. Open the *Project Management* view of the *Process Automation* module.
+. Click on the name of the category of the project.
+. In the *<category name>* view, click the *Finance Analysis* icon image:financeanalysis-icon.png[ symbol,1.5%,1.5%] the panel of the project you want to analyze.
```

Comment:
It looks like this is missing an "of" or "for" before "the panel of the project you want to analyze."

Reply:
... on ...